configfile/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+include_directories(../include/gmapping)
+add_library(configfile configfile.cpp)
```

Comment:
is this library an INI parser ? Why not used the standard boost property tree? http://www.boost.org/doc/libs/1_55_0/doc/html/boost_propertytree/parsers.html#boost_propertytree.parsers.ini_parser